Mathlib/Combinatorics/SimpleGraph/Clique.lean

```diff
@@ -373,6 +377,37 @@ theorem cliqueFree_completeMultipartiteGraph {ι : Type*} [Fintype ι] (V : ι 
   rw [← top_adj, ← f.map_adj_iff, comap_adj, top_adj] at hn
   exact absurd he hn
 
+namespace completeMultipartiteGraph
+
+variable {ι : Type*} (V : ι → Type*)
+
+/-- Embedding of the complete graph on `ι` into `completeMultipartiteGraph` on `ι` nonempty parts.-/
+def topEmbedding (f : ∀ (i : ι), V i) :
+    (⊤ : SimpleGraph ι) ↪g completeMultipartiteGraph V where
+  toFun := fun i ↦ ⟨i, f i⟩
+  inj' := fun _ _ h ↦ (Sigma.mk.inj_iff.1 h).1
+  map_rel_iff' := by simp
+
+theorem notCliqueFree_le_card [Fintype ι] (f : ∀ (i : ι), V i) (hc : n ≤ Fintype.card ι) :
```

Comment:
```suggestion
theorem not_cliqueFree_of_le_card [Fintype ι] (f : ∀ (i : ι), V i) (hc : n ≤ Fintype.card ι) :
```
and similarly below. The idea is that `cliqueFree` is a token, but `notCliqueFree` isn't, instead it's `not` getting applied to `cliqueFree`. And the `of` is saying that `le_card` is one of the assumptions.